src/Page/Page.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+import { ThemeProvider } from '@emotion/react';
+import { StyledPage } from './pageStyles';
+import theme from 'src/styles/theme';
+
+export interface PageProps {
+    /**
+     * These are the classes that would be applied to the Page component.
+     */
+    className?: string;
+}
+
+/**
+ * Page Component
+ * 
+ * The Page Component can be used to render other child components.
+ */
+
+const Page: FC<PageProps> = props =>
+{
+    const { children, className } = props;
```

Comment:
Also, please destructure `children` and `className` in the function call as opposed to in the body of the function as we do in other components.

Reply:
done